release-engineering/role-handbooks/branch-manager.md

```diff
@@ -292,77 +310,146 @@ The following are some ways to determine if the release process was successful:
-   '1.14':
-      CONFIG: '1.14'
-      GO_VERSION: 1.12.1
-      K8S_RELEASE: latest
-      BAZEL_VERSION: 0.21.0
-   '1.13':
-      CONFIG: '1.13'
-      GO_VERSION: 1.11.5
-      K8S_RELEASE: stable-1.13
-    BAZEL_VERSION: 0.18.1
-
-   # Note that the last block i.e. the `1.12` block was removed
+      experimental:
+         CONFIG: experimental
+         GO_VERSION: 1.12.7
+         K8S_RELEASE: stable
+         BAZEL_VERSION: 0.28.1
+         UPGRADE_DOCKER: 'true'
+      master:
+         CONFIG: master
+         GO_VERSION: 1.12.7
+         K8S_RELEASE: stable
+         BAZEL_VERSION: 0.23.2
+      '1.16':
+         CONFIG: '1.16'
+         GO_VERSION: 1.12.7
+         K8S_RELEASE: latest-1.16
+         BAZEL_VERSION: 0.23.2
+      '1.15':
+         CONFIG: '1.15'
+         GO_VERSION: 1.12.7
+         K8S_RELEASE: stable-1.15
+         BAZEL_VERSION: 0.23.2
+      '1.14':
+         CONFIG: '1.14'
+         GO_VERSION: 1.12.5
+         K8S_RELEASE: stable-1.14
+         BAZEL_VERSION: 0.21.0
+      '1.13':
+         CONFIG: '1.13'
+         GO_VERSION: 1.11.5
+         K8S_RELEASE: stable-1.13
+         BAZEL_VERSION: 0.18.1
+
+   # Note that the last block i.e., the `1.12` block was removed
    ```
 
-1.  Create a PR with this change and wait for it to be merged. Then wait for the `test-infra-push-kubekins-e2e` presubmit to finish (you can [check on prow](https://prow.k8s.io/?job=post-test-infra-push-kubekins-e2e)). Pull latest from `master` before continuing.
+4. Create a PR with this change and wait for it to be merged. ([example PR](https://github.com/kubernetes/test-infra/pull/13876))
+
+   **Wait for the `test-infra-push-kubekins-e2e` presubmit to finish (you can [check on prow](https://prow.k8s.io/?job=post-test-infra-push-kubekins-e2e)).**
+
+   **Pull latest from `master` before continuing.**
+
+5. Generate the new release branch jobs:
+
+   ```shell
+   bazel run //experiment:prepare_release_branch
+   ```
+
+6. Search for and update jobs that must be manually bumped
+
+   `grep` for [`manual-release-bump-required`](https://github.com/kubernetes/test-infra/search?q=manual-release-bump-required&unscoped_q=manual-release-bump-required) in [test-infra](https://github.com/kubernetes/test-infra) and duplicate block entries appropriately
+
+7. Update release dashboards in the [Testgrid config](https://git.k8s.io/test-infra/config/testgrids/config.yaml) ([example commit](https://github.com/kubernetes/test-infra/pull/13882/commits/8aad58b09ae718139b44c70512d2d133d094ed82))
+   - Remove the oldest release `sig-release-<version>-{master,informing,all}` dashboards e.g., `sig-release-1.12-all`
+   - Add dashboards for the current release e.g., `sig-release-1.16-{master,informing,all}`
+
+8. Check for and resolve configuration errors ([example commit](https://github.com/kubernetes/test-infra/pull/13882/commits/867e9ee0329228c6c79399dc67097d0f94814f8f)):
+
+   ```shell
+   bazel test //testgrid/cmd/configurator/...
```

Comment:
Ugh why do we not have a single target for this...

You'll want to use `//config/...` to pickup prowjob tests, also where we intend to move testgrid config tests (ref: https://github.com/kubernetes/test-infra/issues/13690)

And `//hack:verify-config` will catch more things, like generated jobs, prow config changes, etc.
```suggestion
   bazel test //config/... //testgrid/... //hack:verify-config
```